charger/tasmota.go

```diff
@@ -47,20 +46,19 @@ func NewTasmotaFromConfig(other map[string]interface{}) (api.Charger, error) {
 }
 
 // NewTasmota creates Tasmota charger
-func NewTasmota(embed embed, uri, user, password string, channel int, standbypower float64, cache time.Duration) (*Tasmota, error) {
-	conn, err := tasmota.NewConnection(uri, user, password, channel, cache)
+func NewTasmota(embed embed, uri, user, password string, channels []int, standbypower float64, cache time.Duration) (*Tasmota, error) {
+	conn, err := tasmota.NewConnection(uri, user, password, channels, cache)
 	if err != nil {
 		return nil, err
 	}
 
 	c := &Tasmota{
-		conn:    conn,
-		channel: channel,
+		conn: conn,
 	}
 
 	c.switchSocket = NewSwitchSocket(&embed, c.Enabled, c.conn.CurrentPower, standbypower)
 
-	return c, c.conn.ChannelExists(channel)
+	return c, c.conn.ChannelExists()
```

Comment:
gehört das nicht (unexported) ins NewConnection? Die kennt die Channels ja schon...

Reply:
Das hatte ich extra nur in den Charger eingebaut, da bei einem Tasmota Meter kein Schaltkanal geprüft werden muss ...

Reply:
Ahh, verstehe 👍🏻